config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root  "menus#show"
+  
+  post  "menus/spin-gacha",       to: "menus#spin_gacha", as: "spin_gacha"
+  post  "menus/toggle-favorite",  to: "menus#toggle_fav", as: "toggle_fav"
+  get   "menus/favorite-menus",   to: "menus#fav_menus",  as: "fav_menus"
+  resources :menus, only: [:index, :show]
```

Comment:
rails は基本的にこの `resources` を使ってルーティングを組み立てたい。
この場合は new と show を分けるとスッキリしそう。

```suggestion
  root to: 'menus#new'
  
  resources :menus, only: [:index, :new, :show, :create] do 
    resources :favorites, only: [:index, :create, :destroy]
  end
```